bundles/org.openhab.binding.verisure/README.md

```diff
@@ -1,12 +1,11 @@
 # Verisure Binding
 
-This is an openHAB binding for Verisure Alarm System, by Securitas Direct.
+This is an openHAB binding for Verisure Smart Alarms by Verisure Securitas.
 
-This binding uses the rest API behind the Verisure My Pages:
+This binding a rest API behind the Verisure My Pages:
```

Comment:
A word is missing I believe: This binding is a ...

Reply:
Fixed!